photon-ml/src/main/scala/com/linkedin/photon/ml/avro/AvroUtils.scala

```diff
@@ -151,7 +151,8 @@ object AvroUtils {
     numPartitions: Int): NameAndTermFeatureSetContainer = {
 
     val nameAndTermFeatureSets = featureSectionKeys.map { featureSectionKey =>
-      (featureSectionKey, AvroUtils.readNameAndTermSetFromGenericRecords(genericRecords, featureSectionKey, numPartitions))
+      (featureSectionKey, AvroUtils.readNameAndTermSetFromGenericRecords(genericRecords,
+        featureSectionKey, numPartitions))
```

Comment:
For me personally, this style is about as unpleasant and esthetically unsatisfactory as cutting up verses as in: "Serait-ce déjà lui ? - C'est bien à l'escalier / Dérobé.". But I've "fixed" it.